conftest.py

```diff
@@ -192,12 +193,25 @@ def setup_artifact_path(item: pytest.Item, artifact_name: str):
     return os.path.join(output_dir, artifact_name)
 
 
-def get_task_files_for_id(task_id: str) -> set:
+def get_task_files_for_id(task_id: str) -> dict:
     try:
-        return set(subprocess.check_output(['dcos', 'task', 'ls', task_id, '--all']).decode().split())
+        # Example output:
+        # drwxr-xr-x  6  nobody  nobody      4096  Jul 21 22:07                             jre1.8.0_144
+        # drwxr-xr-x  3  nobody  nobody      4096  Jun 28 12:50                          libmesos-bundle
+        # -rw-r--r--  1  nobody  nobody  32539549  Jan 04 16:31  libmesos-bundle-1.10-1.4-63e0814.tar.gz
+        ls_lines = subprocess.check_output(['dcos', 'task', 'ls', task_id, '--long', '--all']).decode().split('\n')
+        ret = {}
+        for line in ls_lines:
+            # example elems: ['drwxr-xr-x', '6', 'nobody', 'nobody', '4096', 'Jul 21 22:07', 'jre1.8.0_144']
+            elems = [s.strip() for s in filter(None, s.split('  '))]
```

Comment:
Was this meant to be `line.split('  ')`?

Reply:
It's specifically `'  '` in order to split the elements correctly. Single space would result in the timestamp being fragmented, or in problems with e.g. filenames containing single spaces (hopefully it's unlikely that filenames will contain two spaces).

Reply:
No, I mean the variable `s` doesn't exist so `s.split('  ')` is invalid?

Reply:
Ohh, whoops. Fixed now. Just in case I manually ran the handler again and it's fixed now.

Reply:
Haha, looks like github is eating double spaces: `foo  bar`